cmd/mimir/help-all.txt.tmpl

```diff
@@ -287,6 +287,8 @@ Usage of ./cmd/mimir/mimir:
     	TTL for caching object attributes for chunks. If the metadata cache is configured, attributes will be stored under this cache backend, otherwise attributes are stored in the chunks cache backend. (default 168h0m0s)
   -blocks-storage.bucket-store.chunks-cache.backend string
     	Backend for chunks cache, if not empty. Supported values: memcached.
+  -blocks-storage.bucket-store.chunks-cache.fine-grained-chunks-caching-enabled
+    	[experimental] Enable fine-grained caching of chunks in the store-gateway. This should reduce the required bandwidth and memory utilization.
```

Comment:
```suggestion
    	[experimental] Enable fine-grained caching of chunks in the store-gateway. This reduces the required bandwidth and memory utilization.
```

Reply:
Your wording sounds very confident. We're still not _that_ confident that this always reduces memory and bandwidth. Is there another way to word this?

Reply:
If there is a way for the user to provide feedback, that could be a nice compromise. How about something like, "If this is not what you experience, contact ..." or something to that effect?

Reply:
we spoke offline and agreed to keep it as `reduces` since this is already an experimental feature and it's implied it may not work properly. We're also planning to just remove this option soon - not unlikely before the next stable release.